std/assembly/internal/arraybuffer.ts

```diff
@@ -34,7 +34,7 @@ export function allocateUnsafe(byteLength: i32): ArrayBuffer {
   return changetype<ArrayBuffer>(buffer);
 }
 
-export function reallocateUnsafe(buffer: ArrayBuffer, newByteLength: i32): ArrayBuffer {
+export function reallocateUnsafe(buffer: ArrayBuffer, newByteLength: i32, zeroFill: bool = false): ArrayBuffer {
```

Comment:
Sorry, if we don't want break existing behaviours `zeroFill` should be `true` by default

Reply:
I was following your example as you presented it. I'll make the fix now.

Reply:
Yeah, my bad